src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    height: 100%;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    height: 100%;
+    align-items: center;
+    font-weight: 500;
```

Comment:
This is the default value. No need to write it
```suggestion
```

Reply:
The default value is 400, isn`t it?